src/components/SQForm/SQFormTextarea.tsx

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import TextField from '@material-ui/core/TextField';
 import Grid from '@material-ui/core/Grid';
-import {useFormikContext} from 'formik';
-
+import {useFormikContext, FormikProps} from 'formik';
 import {useForm} from './useForm';
+import {TextareaAutosizeProps, TextFieldProps} from '@material-ui/core'
+import BaseFieldProps from '../../types/BaseFieldProps';
 import {toKebabCase} from '../../utils';
 
+interface SQFormTextareaProps extends BaseFieldProps {
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder: string;
+  /** Disabled property to disable the input if true */
+  isDisabled: boolean;
+  /** Required property used to highlight input and label if not fulfilled */
+  isRequired: boolean;
+  /** Custom onBlur event callback */
+  onBlur: React.FocusEventHandler;
+  /** Custom onChange event callback */
+  onChange: React.ChangeEventHandler;
+  /** Number of rows to display when multiline option is set to true. */
+  rows: number;
+  /** Maximum number of rows to display when multiline option is set to true. */
+  rowsMax: number;
+  /** Attributes applied to the `textarea` element */
+  inputProps: TextareaAutosizeProps;
```

Comment:
I don't think the type for inputProps should be TextAreaAutosizeProps because we are not using TextAreaAutosize component inside SQFormTextArea component. I think the type should be `React.InputHTMLAttributes<HTMLInputElement>`  as the component being used is TextField component but would like to see what others say in the matter. @SeanGroff @codyarose

Reply:
@sohail-panezai is correct; we shouldn't use `TextAreaAutosizeProps` because we're not using that component here, and the type for `inputProps` should be the html textarea element's props/attributes not a React component's props.
`React.InputHTMLAttributes<HTMLTextAreaElement>` should work